bin/plot_krun_results

```diff
@@ -199,6 +213,8 @@ def main(is_interactive, data_dcts, plot_titles, window_size, outfile,
         all_changepoint_means.append(page_changepoint_means)
     else:  # Create multiple pages.
         for key in sorted(data_dcts['data']):
+            if key.split(":")[0] == "empty":
```

Comment:
Kind of. It's the hack which prevents the plots crashing for the empty benchmarks.